frontend/src/data/config/MetricConfig.ts

```diff
@@ -252,4 +254,25 @@ export const METRIC_CONFIG: Record<string, VariableConfig[]> = {
       },
     },
   ],
+  health_insurance: [
+    {
+      variableId: "health_coverage",
+      variableDisplayName: "Coverage",
+      metrics: {
+        count: {
+          metricId: "with_health_insurance",
+          fullCardTitleName: "Count of Health Insurance",
```

Comment:
I think this copy, which is what shows up on the table card as the column header, could be clearer. Maybe "Count of individuals with health insurance" or "Individuals with health insurance"? You could raise it with team.

Reply:
Done :)